newdle/client/src/components/ParticipantTable.js

```diff
@@ -75,6 +72,36 @@ function AvailabilityRow({
   const duration = useSelector(getNewdleDuration);
   const startTime = toMoment(startDt, 'YYYY-MM-DDTHH:mm');
 
+  const isMobile = useIsMobile();
+  const renderParticipants = () => {
+    if (!numberOfParticipants) {
+      return 'There are no participants yet.';
+    }
+    if (participants.length === 0) {
+      return 'Nobody has voted yet.';
+    }
+    if (availableCount === 0) {
+      return 'Nobody is available.';
+    }
+    if (!isMobile || active) {
+      return <ParticipantNames participants={participants} />;
+    }
+    return null;
+  };
+
+  const renderParticipantLabel = () => {
+    if (availableCount > 0 && (!isMobile || !active || finalized)) {
```

Comment:
Together with my comment above I would remove the `availableCount` condition here and always show a label in the mobile view, but use a red one with a different icon in case nobody is available.

Reply:
Also, better to structure it like this:

```js
if (...) {
  return null;
}
return ...;
```

Reply:
If we always want to show a label, then the if condition can be removed. Isn't that what you meant?

Reply:
I played around a bit with the label and honestly I'm not sure when it is actually useful... we always have the circle which already shows this information. But I'd leave it as it is for now, this can be revisited later if it turns out to be wasting space or people dislike it...